src/main/steps/applicant1/how-did-you-change-your-name/content.ts

```diff
@@ -35,44 +33,54 @@ const cy: typeof en = ({ isDivorce }) => ({
   anotherWayMoreDetails:
     "Rhowch fanylion pryd a sut y gwnaethoch newid eich enw. Gofynnir i chi uwchlwytho llun neu sgan o'r dogfennau sy'n profi eich bod wedi newid eich enw yn ddiweddarach yn y cais hwn, neu gallwch eu hanfon drwy'r post. Os nad oes gennych unrhyw ddogfennau, eglurwch pam yma.",
   errors: {
-    applicant1ChangedNameHow: {
+    applicant1NameChangedHow: {
       required: 'Nid ydych wedi ateb y cwestiwn. Mae angen i chi ddewis ateb cyn parhau.',
-    },
-    applicant1ChangedNameHowAnotherWay: {
-      required:
+      applicant1ChangedNameHowAnotherWay:
         'Rydych wedi dweud eich bod wedi newid eich enw mewn ffordd arall ond heb ddarparu manylion. Rhowch fanylion am sut y gwnaethoch newid eich enw.',
     },
   },
 });
 
 export const form: FormContent = {
   fields: {
-    applicant1ChangedNameHow: {
-      type: 'radios',
-      classes: 'govuk-radios',
+    applicant1NameChangedHow: {
+      type: 'checkboxes',
       label: l => l.title,
       labelHidden: true,
+      validator: atLeastOneFieldIsChecked,
       values: [
-        { label: l => l.sendingOffMarriageCertificate, value: ChangedNameHow.MARRIAGE_CERTIFICATE },
         {
+          name: 'applicant1NameChangedHow',
+          label: l => l.sendingOffMarriageCertificate,
+          value: ChangedNameHow.MARRIAGE_CERTIFICATE,
+        },
+        {
+          name: 'applicant1NameChangedHow',
           label: l => l.deedPoll,
           value: ChangedNameHow.DEED_POLL,
           conditionalText: l => `<p class="govuk-label">${l.deedPollMoreDetails}</p>`,
         },
         {
+          name: 'applicant1NameChangedHow',
           label: l => l.anotherWay,
           value: ChangedNameHow.OTHER,
           subFields: {
             applicant1ChangedNameHowAnotherWay: {
               type: 'textarea',
               label: l => l.anotherWayMoreDetails,
               labelSize: null,
-              validator: isFieldFilledIn,
```

Comment:
i think this is a better way but due to nightly build failing, its best to merge this and clean it afterwards.